docs/source/serenata_toolbox.rst

```diff
@@ -1,41 +1,14 @@
 serenata_toolbox package
 ========================
 
-Submodules
-----------
+Subpackages
```

Comment:
Changed automagically! 
So, I can change it manually, but it was generated, so I don't know.